packages/wallet/.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+// NOTE: This package has inconsistent use of trailing commas and bracket spacing
+
+module.exports = {
+    ...require('../../.prettierrc.js'),
+    "printWidth": 120,
+    "singleQuote": false,
+    "bracketSpacing": false,
```

Comment:
What's the reason for using these values?

I, for one, prefer a shorter width (80) and that we have `bracketSpacing` on.

Reply:
Me too; this is just how it is now.